app/src/main/res/layout/activity_login.xml

```diff
@@ -11,8 +11,7 @@
     <ImageView
         android:layout_width="144dp"
         android:layout_height="144dp"
-        android:src="@drawable/ic_launcher_background"
```

Comment:
For files that are likely to be worked upon by other members, don't commit. You can keep this change on your local machine, then create a separate changelist, add this file to that changelist, and commit using other changelists.